lambda_src/lambda_function.py

```diff
@@ -155,26 +155,29 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
             'statusCode': 200,
             'body': b64encode(compress(data_dumps.encode())).decode(),
             'isBase64Encoded': True,
-            'headers': {'Content-Encoding': 'gzip'}
+            'headers': {'Content-Encoding': 'gzip'},
         }
 
-    if len(response) > 6_000_000:
-        response = dumps(
-            {
-                'data': [
-                    [
-                        rn,
-                        {
-                            'error': (
-                                f'Response size ({len(response)} bytes) will likely'
-                                'exceeded maximum allowed payload size (6291556 bytes).'
-                            )
-                        },
+    if len(dumps(response)) > 6_000_000:
+        response = {
+            'statusCode': 200,
+            'body': dumps(
+                {
+                    'data': [
+                        [
+                            rn,
+                            {
+                                'error': (
+                                    f'Response size ({len(dumps(response))} bytes) will likely '
```

Comment:
so as to not do `len(dumps(...))` twice, let's put it in a variable `response_length`

with the exact constant, we can remove the word "likely" here

Reply:
> with the exact constant, we can remove the word "likely" here

Removed it  👍 

> so as to not do len(dumps(...)) twice, let's put it in a variable response_length

This actually turns into a function in [#59](https://github.com/Snowflake-Labs/geff/pull/59/files#diff-2b9d704e4645a913e70521e3fa194921702cf9e9aee4ce6573403adcd80c2d58R225) so it might have to be repeated I suppose?

Reply:
In #59 you can pass the integer length instead of the string, as well.

Reply:
Oo right, made the change here and tested it again, okay to merge?